### DISCUSSION TITLE
I need screenshot  in binary buffer, Not to save locally

### DISCUSSION THREAD

**Discussion by suredarv06** [seleniumbase/SeleniumBase#2647] [Q&A] [answered]
Hi,
I need to take screenshot and the I need result in binary buffer, so that it will be helpful for my task

**Comment by mdmintz** [comment, marked as answer]
After you find an element, you can use:

```
element.screenshot_as_base64
element.screenshot_as_png
```

Use `"body"` for the `selector` when finding an element if you want a screenshot of the whole page.

**Reply by suredarv06** [reply]
Superb , But I want the entire page screenshot as binary file

**Reply by mdmintz** [reply]
Did using `"body"` for the selector not work? SeleniumBase methods just uses raw Selenium for saving the screenshots. It doesn't create its own internal tools for screenshots.

**Reply by suredarv06** [reply]
While selecting body element , I couldn't get the full page screenshot , the captured screenshot is partial . I have used headless mode  too. some portion at the bottom is missing.
Is there any way to get it in full size of the page?

**Reply by mdmintz** [reply]
Make the window size bigger:

```python
self.set_window_size(width, height)
```

**Reply by suredarv06** [reply]
Hi mintz,


https://github.com/seleniumbase/SeleniumBase/discussions/2647#discussion-6435679

I need some documentation, about elements , like element.screenshot_as_base64  ,want to explore other functions and properties of elements  ,Could you help to find document?

**Reply by mdmintz** [reply]
https://github.com/SeleniumHQ/selenium
